pilot/pkg/security/authz/matcher/header.go

```diff
@@ -55,6 +57,37 @@ func HeaderMatcher(k, v string) *routepb.HeaderMatcher {
 	}
 }
 
+// HeaderMatcherWithRegex converts a key, value string pair to a corresponding
+// HeaderMatcher to support matching the value presented in the inline header with
+// multiple values concatenated by commas, as per RFC7230.
+func HeaderMatcherWithRegex(k, v string) *routepb.HeaderMatcher {
```

Comment:
Can we call this something else so it doesn't sound like its a generic header match

Reply:
any suggestion?

Reply:
Updated to InlineHeaderMatcher, let me know if it's better.